README.md

```diff
@@ -80,8 +80,7 @@ Custom attr:
  * `pstsPaddingMiddle` If true, the tabs start at the middle of the view (Like Newsstand google app)
  * `pstsTextStyle` Set the text style, default bold
  * `pstsTextSelectedStyle` Set the text style of the selected tab, default bold
- * `pstsTextAlpha` Set the text alpha transparency, default 0.5
- * `pstsTextSelectedAlpha` Set the text alpha transparency of the selected tab, default 1
+ * `pstsTextColorInactive` Set text color of inactive text so you can use ARGB without using `View.setAlpha`
```

Comment:
:+1: for the spirit of the change, but I would flip this so you specify `pstsTextColorSelected` (with `textColor` being used for the "inactive" state) which would match the other attributes.